discovery-provider/src/trending_strategies/trending_strategy_factory.py

```diff
@@ -22,7 +22,7 @@
 )
 
 DEFAULT_TRENDING_VERSIONS = {
-    TrendingType.TRACKS: TrendingVersion.aSPET,
+    TrendingType.TRACKS: TrendingVersion.ML51L,
```

Comment:
why is this still aSPET if ML51L changes are already out?

Reply:
It shouldn't really matter bc the client decides which trending version it receives via an optimizely flag - which it uses to pass as an url param to get the trending tracks from discovery. 
Usually we want to test that it's consistent and working as expected before switching over the default.

Reply:
I see. When I was running client on my remote it used the default so I changed it here. Probably safer to revert this.